src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -42,9 +41,34 @@ public function getAccount() {
    * Set the current object account.
    *
    * @param AccountInterface $account
+   *   The user object.
+   * @return $this
    */
-  public function setAccount(AccountInterface $account) {
+  public function setAccount( $account) {
     $this->currentUser = $account;
+    return $this;
+  }
+
+  /**
+   * Get the field configuration.
+   *
+   * @param $key
+   *   Specific key from the configuration. Optional.
+   *
+   * @return array|string
+   *   Value of a configuration or all the configurations.
+   */
+  public function getConfiguration($key = NULL) {
+    return empty($this->configuration[$key]) ? $this->configuration : $this->configuration[$key];
```

Comment:
IMO this is not a good thing to do, if you pass in a key that doesn't exist, you will get all configuration. That's a bit of a WTF. We should check isset($key) first before returning all configuration.